src/main/java/com/iemr/hwc/fhir/provider/condition/ConditionExtProvider.java

```diff
@@ -0,0 +1,62 @@
+package com.iemr.hwc.fhir.provider.condition;
+
+import ca.uhn.fhir.rest.annotation.Create;
+import ca.uhn.fhir.rest.annotation.ResourceParam;
+import ca.uhn.fhir.rest.api.MethodOutcome;
+import ca.uhn.fhir.rest.server.IResourceProvider;
+import com.iemr.hwc.fhir.model.condition.ConditionExt;
+import com.iemr.hwc.fhir.service.condition.ConditionService;
+import org.hl7.fhir.instance.model.api.IBaseResource;
+import org.hl7.fhir.r4.model.CodeableConcept;
+import org.hl7.fhir.r4.model.Coding;
+import org.hl7.fhir.r4.model.OperationOutcome;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.stereotype.Component;
+import javax.servlet.http.HttpServletRequest;
+import java.util.ArrayList;
+import java.util.List;
+
+@Component
+public class ConditionExtProvider implements IResourceProvider {
+
+    @Autowired
+    private ConditionService conditionService;
+
+    @Override
+    public Class<? extends IBaseResource> getResourceType() {
+        return ConditionExt.class;
+    }
+
+    public ConditionExt getMockCondition(){
+        ConditionExt mockCondition = new ConditionExt();
```

Comment:
What is the use of mockCondition?

Reply:
Not of any use while syncing to the server. To be used as a template resource while down-syncing.